Mathlib/Probability/Kernel/Composition.lean

```diff
@@ -1096,6 +1100,27 @@ lemma const_comp' (μ : Measure γ) (κ : Kernel α β) [IsMarkovKernel κ] :
     const β μ ∘ₖ κ = const α μ := by
   ext; simp_rw [const_comp, measure_univ, one_smul, const_apply]
 
+lemma map_comp (κ : Kernel α β) (η : Kernel β γ) {f : γ → δ} (hf : Measurable f) :
```

Comment:
I think you can drop measurability of `f` here: otherwise, both sides are zero thanks to `map_of_not_measurable`.

Reply:
Thanks! That code was written before the map refactor and I forgot to adapt.
I removed the assumption, and added two simp lemmas `zero_comp` and `comp_zero` because the new proof made me see they were missing.